assets/src/blocks/Happypoint/useHappypointImageData.js

```diff
@@ -0,0 +1,32 @@
+import {useState, useEffect} from '@wordpress/element';
+import {fetchJson} from '../../functions/fetchJson';
+import {addQueryArgs} from '../../functions/addQueryArgs';
+
+const {apiFetch} = wp;
+
+export const useHappypointImageData = imageId => {
+  const [imageData, setImageData] = useState({});
+
+  useEffect(() => {
+    const loadImageData = async () => {
+      try {
+        const args = {
+          id: imageId,
+        };
+
+        const baseUrl = document.body.dataset.nro;
+
+        const data = baseUrl ?
+          await fetchJson(`${baseUrl}/wp-json/${addQueryArgs('planet4/v1/get-happypoint-data', args)}`) :
```

Comment:
We register this rest route in the [blocks repo](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/e2bad1093eea5e03911fa752c20de5d1ebabcc1f/classes/rest/class-rest-api.php#L222), we should probably move it to the theme?